app/components/user_avatars_stack/index.tsx

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {useIntl} from 'react-intl';
+import {Platform, StyleProp, Text, TouchableOpacity, View, ViewStyle} from 'react-native';
+
+import FormattedText from '@components/formatted_text';
+import {useTheme} from '@context/theme';
+import {useIsTablet} from '@hooks/device';
+import {bottomSheet} from '@screens/navigation';
+import {preventDoubleTap} from '@utils/tap';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+import UserAvatar from './user_avatar';
+import UsersList from './users_list';
+
+import type UserModel from '@typings/database/models/servers/user';
+
+const OVERFLOW_DISPLAY_LIMIT = 99;
+
+type Props = {
+    currentUserId: string;
+    users: UserModel[];
+    breakAt?: number;
+    style?: StyleProp<ViewStyle>;
+    teammateNameDisplay: string;
+}
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
+    const size = 24;
+
+    let STATUS_BUFFER = Platform.select({
```

Comment:
We only work with two platforms, so you can do `Platform.select({ios: 3, default: 2})`. That way STATUS_BUFFER won't be undefined.